addons/portal/models/mail_message.py

```diff
@@ -16,8 +16,10 @@ def portal_message_format(self):
 
     def _portal_message_format(self, fields_list):
         vals_list = self._message_format(fields_list)
+        message_subtype_note_id = self.env['ir.model.data'].xmlid_to_res_id('mail.mt_note')
         IrAttachmentSudo = self.env['ir.attachment'].sudo()
         for vals in vals_list:
+            vals['is_message_subtype_note'] = message_subtype_note_id and vals.get('subtype_id', [False])[0] == message_subtype_note_id
```

Comment:
Almost there, the commit message still mentions adding a "is_note" param ^^

As we made a lot of changes, not sure if this needs to be re-tested by Luc, I'm guessing you re-tested thoroughly?